Mail/Views/New Message/ComposeMessageToolbarModifier.swift

```diff
@@ -39,7 +39,7 @@ struct ComposeMessageToolbarModifier: ViewModifier {
             .navigationBarTitleDisplayMode(.inline)
             .toolbar {
                 ToolbarItem(placement: .navigationBarLeading) {
-                    if !platformDetector.isMac {
+                    if !platformDetector.isMac || Bundle.main.isExtension {
```

Comment:
There is a component that uses DI to do just this.
I'd prefer we use this single source of truth for now instead of checking `Bundle.` directly.

```swift
@LazyInjectService private var platformDetector: PlatformDetectable
let isInExtension = platformDetector.isInExtension
```

On kDrive you can use `AppContextService` for this. 
It is better as it does not rely on Bundle at all and knows exactly which extension is in use (share / notifications …)
We should bring it to Mail at some point, you could if you want 🙌 
```swift
@LazyInjectService var appContextService: AppContextServiceable
```